src/passes/Precompute.cpp

```diff
@@ -87,6 +89,12 @@ class StandaloneExpressionRunner : public ExpressionRunner<StandaloneExpressionR
     return Flow(NONSTANDALONE_FLOW);
   }
   Flow visitGetGlobal(GetGlobal *curr) {
+    auto global = module->getGlobalOrNull(curr->name);
```

Comment:
`auto*` (with `*`, that ensures it's a pointer type, in the unlikely chance the function is later changed to return something else)